doc/content/source/solvers/MFEMOperatorJacobiSmoother.md

```diff
@@ -0,0 +1,15 @@
+# MFEMOperatorJacobiSmoother
+
+## Summary
+
+!syntax description /Solver/MFEMOperatorJacobiSmoother
+
+## Overview
+
+Defines and builds an `mfem::OperatorJacobiSmoother` solver to use to perform Jacobi iterations on the MFEM equation system.
```

Comment:
Maybe it's worth it to mention that it's generally supposed to be used as a preconditioner? Also perhaps that it's compatible with partialy-assembled systems